CONTRIBUTING.md

```diff
@@ -57,14 +57,16 @@ If there are any errors during the compilation please check our public build sta
 
 ### Importing to IDE
 
-We use IntelliJ Idea IDE for development.
+We use IntelliJ IDEA IDE for development.
 
-Generate IntelliJ Idea project files
-```
-> gradlew assemble idea 
-```
+In order to work on CUBA it has to imported as a project in IntelliJ IDEA. To do that execute the following steps:
```

Comment:
Did you mean `has to BE imported` ?

Reply:
yes - fixed. Funny how small a change can possibly and still make errors in it... :)